server/backend/backend_test.go

```diff
@@ -3,25 +3,38 @@
 package backend
 
 import (
+	"fmt"
 	"math/big"
+	"os"
 	"testing"
 	"time"
 
+	"github.com/blendle/zapdriver"
 	"github.com/gochain-io/explorer/server/models"
 
 	"github.com/gochain-io/gochain/v3/common"
 	"github.com/gochain-io/gochain/v3/core/types"
 	"github.com/gochain-io/gochain/v3/rlp"
-	"github.com/rs/zerolog/log"
 )
 
-var testBackend = NewBackend("127.0.0.1:27017", "https://rpc.gochain.io", "testdb", nil, nil)
+var testBackend *Backend
+
+func TestMain(m *testing.M) {
+	lgr, err := zapdriver.NewDevelopment()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to create logger: %v\n", err)
+		os.Exit(1)
+	}
+	testBackend = NewBackend("127.0.0.1:27017", "https://rpc.gochain.io", "testdb", nil, nil, lgr)
+	code := m.Run()
+	os.Exit(code)
+}
```

Comment:
zap has `package zaptest` for test loggers: `zaptest.NewLogger(t)`

Using it will require having one `*Backend` per test, instead of a global var, but that is more idiomatic and safer anyways.